packages/CalDAV-Core.package/CALApp.class/instance/buildAddCalendarButton..st

```diff
@@ -1,9 +1,11 @@
 toolbuilder
 buildAddCalendarButton: aBuilder
-	
+
+	| anchorToImage |
```

Comment:
Leerzeile nach Deklaration der temporären Variablen fehlt (in sehr vielen folgenden Methoden auch)